static/app/views/explore/hooks/useDeleteQuery.tsx

```diff
@@ -2,18 +2,24 @@ import {useCallback} from 'react';
 
 import useApi from 'sentry/utils/useApi';
 import useOrganization from 'sentry/utils/useOrganization';
+import {useInvalidateSavedQueries} from 'sentry/views/explore/hooks/useGetSavedQueries';
 
 export function useDeleteQuery() {
   const api = useApi();
   const organization = useOrganization();
+  const invalidateSavedQueries = useInvalidateSavedQueries();
```

Comment:
Updated a bunch of hooks to invalidate the GET saved queries endpoint key so that UI updates to reflect new saved queries state on update